Classes/Settings/Defaults+ExternalLinks.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Defaults+ExternalLinks.swift
+//  Freetime
+//
+//  Created by Ivan Smetanin on 08/12/2018.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension UserDefaults {
+
+    private static let defaultKey = "com.whoisryannystrom.freetime.is-need-open-external-links-in-safari"
```

Comment:
Why don't we name this `should-open-external-links-in-safari` too? Not that big of a deal of course, just wondering why.

Reply:
Forgot to fix